examples/trainer_keras.py

```diff
@@ -4,13 +4,18 @@
 import matplotlib.pyplot as plt
 import numpy as np
 
-from data import dummy_data_generator, get_m4_data
+from data import dummy_data_generator_multivariate, get_m4_data_multivariate, get_nrj_data, get_kcg_data
 from nbeats_keras.model import NBeatsNet
 
 
+GENERIC_BLOCK = 'generic'
```

Comment:
No need to define them here just use NBeatsNet.GENERIC_BLOCK for example

Reply:
Updated